docs/src/index.md

```diff
@@ -147,20 +147,127 @@ F-test: 2 models fitted on 50 observations
 ## Methods applied to fitted models
 
 Many of the methods provided by this package have names similar to those in [R](http://www.r-project.org).
+- `adjr2`:  adjusted R² for a linear model (an alias for `adjr²`)
+- `aic`: Akaike's Information Criterion, defined as ``-2 \\log L + 2k``, with ``L`` the likelihood of the model, and `k` it the number of consumed degrees of freedom
+- `aicc`: corrected Akaike's Information Criterion for small sample sizes (Hurvich and Tsai 1989)
+- `bic`: Bayesian Information Criterion, defined as ``-2 \\log L + k \\log n``, with ``L``
+the likelihood of the model, ``k`` is the number of consumed degrees of freedom
```

Comment:
```suggestion
- `bic`: Bayesian Information Criterion, defined as ``-2 \\log L + k \\log n``, with ``L`` the likelihood of the model, ``k`` is the number of consumed degrees of freedom
```

Reply:
I'd rather not give the definition here. This is just intended to list features.
```suggestion
- `bic`: Bayesian Information Criterion
```